cdk/typescript/lib/the-lambda-power-tuner-stack.ts

```diff
@@ -0,0 +1,26 @@
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import * as sam from 'aws-cdk-lib/aws-sam';
+
+export class TheLambdaPowerTunerStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    let powerValues = '128,256,512,1024,1536,3008';
+    let lambdaResource = "*";
```

Comment:
I would suggest that we include (commented out) all the possible parameters. Similar to what we do here for the CloudFormation template: https://github.com/alexcasalboni/aws-lambda-power-tuning/blob/master/scripts/deploy-sar-app.yml#L13

Reply:
Can you please make the indentation consistent here?